airflow/providers/ssh/operators/ssh.py

```diff
@@ -63,7 +63,10 @@ class SSHOperator(BaseOperator):
     """
 
     template_fields: Sequence[str] = ("command", "environment", "remote_host")
-    template_ext: Sequence[str] = (".sh",)
+    template_ext: Sequence[str] = (
+        ".sh",
+        ".bash",
+    )
```

Comment:
This looks unrelated? I’m not against its addition, but it should be done in a separate PR.